packages/pytest-simcore/src/pytest_simcore/docker_registry.py

```diff
@@ -231,7 +231,7 @@ def jupyter_service(docker_registry: str, node_meta_schema: Dict) -> Dict[str, A
     )
 
 
-@pytest.fixture(scope="session", params=["2.0.2"])
+@pytest.fixture(scope="session", params=["2.0.3"])

```

Comment:
backwards compatiity not needed here?

Reply:
No, this is just an upgrade, an extra feature. Also while not used it is tested by one of the cases.